api/app/invitationletter/generator.py

```diff
@@ -1,9 +1,105 @@
+from mailmerge import MailMerge
+from app import LOGGER
+from config import GCP_CREDENTIALS_DICT
+from config import GCP_BUCKET_NAME
+import json
+from google.cloud import storage
+import os
+from app.utils import emailer
+from app.utils import pdfconvertor
+from app.events.models import Event
+from app import db
+from app.utils import errors
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = json.dumps(GCP_CREDENTIALS_DICT)
+
+
+OFFER_EMAIL_BODY = """
+Dear {user_title} {first_name} {last_name},
+
+Congratulations! You've been selected to attend the {event_name}!
+
+Please see the attached document below for your acceptance of offer: {host}/offer
```

Comment:
I'm not sure if this is github being weird, but I gave specific copy for this
It's not an offer email. it's an invitation letter